homeassistant/components/tilt_pi/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "tilt_pi",
+  "name": "Tilt Pi",
+  "codeowners": ["@michaelheyman"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/tilt_pi",
+  "homekit": {},
+  "iot_class": "local_polling",
+  "quality_scale": "bronze",
+  "requirements": [],
+  "ssdp": [],
+  "zeroconf": []
```

Comment:
remove empty fields

Reply:
Removed in https://github.com/home-assistant/core/pull/139726/commits/d558469c110131c41507b96e4eb9782d77bcf0d2